apps/demos/src/app/features/template/rx-let/let-template-binding/let-template-binding.component.spec.ts

```diff
@@ -0,0 +1,111 @@
+import { StaticProvider } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterTestingModule } from '@angular/router/testing';
+import { RX_RENDER_STRATEGIES_CONFIG } from '@rx-angular/cdk/render-strategies';
+import { RxState } from '@rx-angular/state';
+
+import { LetTemplateBindingComponent } from './let-template-binding.component';
+import { LetTemplateBindingModule } from './let-template-binding.module';
+
+const RX_ANGULAR_TEST_PROVIDER: StaticProvider = {
+  provide: RX_RENDER_STRATEGIES_CONFIG,
+  useValue: {
+    primaryStrategy: 'native',
+  },
+};
```

Comment:
@rx-angular/core-team do we want to provide this out of the box in the `cdk`?

Reply:
yes please, we should provide a testing package

Reply:
@edbzn we probably will have a testing package soon: https://github.com/rx-angular/rx-angular/pull/1305 :)